cf/src/main/scala/com/timeout/scalacloudformation/Template.scala

```diff
@@ -0,0 +1,12 @@
+package com.timeout.scalacloudformation
+
+import io.circe.Json
+
+
+case class Template(Description: Option[String] = None,
+                    Metadata: Option[Json] = None,
+                    Mappings: Option[Json] = None,
```

Comment:
Yes, and it corresponds to the arity of FindInMap

http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-findinmap.html